web/controllers/forgotpass_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Skillswheel.ForgotpassController do
+  use Skillswheel.Web, :controller
+
+  def index(conn, _params) do
+    render conn, "index.html"
+  end
+
+  def create(conn, %{"forgotpass" => %{"email" => email}}) do
+    email
+    |> Skillswheel.Email.welcome_text_email()
```

Comment:
we could maybe change the name of `welcome_text_email()` to `forgotten_password_email()` just so that it's clear which one it's doing